packages/vuetify/src/composables/position.ts

```diff
@@ -0,0 +1,46 @@
+// Utilities
+import { computed } from 'vue'
+import { convertToUnit } from '@/util/helpers'
+import propsFactory from '@/util/propsFactory'
+
+// Types
+export interface PositionProps {
+  bottom?: boolean | string
+  left?: boolean | string
+  position?: string
```

Comment:
i guess it should be
```suggestion
  position?: 'static' | 'relative' | 'fixed' | 'absolute' | 'sticky'
```